public_app/pages/MoviesDetails.js

```diff
@@ -0,0 +1,114 @@
+import React    from 'react';
+import BigImg   from '../components/BigImg';
+import Nav      from '../components/Nav';
+import Details  from '../components/Details';
+import LogOut   from '../components/LogOut';
+import axios    from 'axios';
+var Link = require ('react-router-dom').Link;
+
+
+
+
+class MoviesDetails extends React.Component {
+
+    constructor (props) {
+        super(props);
+        this.state = {
+            item: {},
+            chat: [],
+            chatInput: {
+                author: '',
+                comment: ''
+            }
+        };
+        console.log(this.state);
+        axios.get(`/api/movies/${this.props.match.params.id}`
```

Comment:
It would be wise to seperate all the `axios` calls in seperate "getData" service file and then call functions from there like in this case

```
// getData service file
export const getMovies(id, callback) {
    // axios request
}
```

```
// MoviesDetails

getMovies(this.props.match.params.id, response => {
    this.setState({
        item: response.data
    });
});

```

that way you could replace the request logic in one place if neccessary